.github/workflows/rust.yml

```diff
@@ -34,6 +34,8 @@ jobs:
     - name: Run tests
       # Profile "ci" is configured in .config/nextest.toml
       run: cargo nextest run --workspace --profile ci
+    - name: Build Debian package
+      run: dpkg-buildpackage -us -uc -b
```

Comment:
*.deb building is not supposed to be part of test pipeline, please take a reference from other binary files building steps